oio/cli/admin/service_rebuild.py

```diff
@@ -124,17 +124,41 @@ class Meta2Rebuild(ServiceRebuildCommand):
     log = getLogger(__name__ + '.Meta2Rebuild')
     columns = ('Reference', 'Status')
     rebuilder_class = Meta2Rebuilder
-    success = False
+    rebuilder = None
+
+    def get_parser(self, prog_name):
+        parser = super(Meta2Rebuild, self).get_parser(prog_name)
+
+        # input
+        parser.add_argument(
+            'service_id',
+            metavar='<service_id>',
+            help='ID of the service to rebuild')
+        # common
+        parser.add_argument(
+            '--rdir-fetch-limit', type=int,
+            help='Maximum of entries returned in each rdir response. '
```

Comment:
Maximum number

and the default is not set

Reply:
I chose to trust the default value chosen by the tool.
https://github.com/open-io/oio-sds/pull/1748/commits/0b23e4849bac7eefcfb69f507ea476e8ed6e3521#diff-c305d063edb11e4180028447625d07e4R42